protobuf/mayastor.proto

```diff
@@ -500,6 +501,26 @@ message RebuildStatsReply {
   uint64 tasks_active = 7; // number of current active tasks
 }
 
+message RebuildHistoryRequest {
+  string uuid = 1;  // uuid of the nexus
+}
+
+message RebuildHistoryReply {
+  string nexus = 1;                          // Name of the nexus
+  repeated RebuildHistoryRecord records = 2; // List of all rebuild records
+                                             // across all the replicas
+}
+
+message RebuildHistoryRecord {
+  string dst_uri = 1;            // URI of rebuilt replica (destination)
+  string src_uri = 2;            // URI of source replica
+  string state = 3;              // Final state of rebuild job
+  uint64 rebuilt_data_size = 4;  // Size of data rebuilt
+  bool   is_partial = 5;         // Full or Partial rebuild
+  string  started_at = 6;        // Start timestamp(UTC) of rebuild job
```

Comment:
Doesn't protobuf have a timestamp type?

Reply:
It indeed has it from "import "google/protobuf/timestamp.proto"", but I couldn't figure a sane way to fill it in using rust datatypes. Any suggestions or samples?

Reply:
I checked by adding a dummy field, looks like this in generated code:

```
    pub ended_at: ::prost::alloc::string::String,
    #[prost(message, optional, tag = "8")]
    **pub some_timestamp: ::core::option::Option<::prost_types::Timestamp>**,
```

Reply:
I think you can use the From/Into trait to convert that `Timestamp` to/from the rust native `SystemTime`